packages/server/api.js

```diff
@@ -661,12 +664,12 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
     (req, res) => uploadSingleImage(req, res, err => {
       if (err) {
         res.status(500).end(JSON.stringify({
-          error: err.message
+          error: Object.assign(errors.UPLOAD_FAILED, {message: err.message})
```

Comment:
`Object.assign` mutates its first argument, doesn't it? Fix pls.

Reply:
Ooops! I meant to do `Object.assign({}, errors.UPLOAD_FAILED, {message: err.message})`. Thanks.